### DISCUSSION TITLE
How to do that dbt models alarms are generated individually instead of relying on the entire job?

### DISCUSSION THREAD

**Discussion by klook-tech-cyrillu** [dagster-io/dagster#22295] [Q&A]
How can I send an email alarm to the owner in the meta configured for each model in my dbt model when a task fails.

Currently, all alarms are generated through the entire job configuration @failure_hook, which cannot meet the requirements because there may be multiple user-defined dbt models in the same job, and the failed model needs to report alarms only to stakeholders, not to job users.

plz help me. thx a lot

**Comment by garethbrickman** [comment]
[This documentation shows how to customize owners of dbt artifacts. ](https://docs.dagster.io/integrations/dbt/reference#customizing-owners). At this time, sending asset alerts to asset owners[ is a Dagster+ Pro tier feature only](https://docs.dagster.io/dagster-plus/managing-deployments/alerts#supported-notification-services:~:text=If%20using%20Dagster%2B%20Pro%2C%20this%20type%20of%20alert%20also%20allows%20you%20to%20send%20alerts%20to%20asset%20owners.).
